lib/improver/tests/ensemble_calibration/ensemble_calibration/helper_functions.py

```diff
@@ -70,6 +70,22 @@ def assertEMOSCoefficientsAlmostEqual(self, first, second):
          """
         self.assertArrayAlmostEqual(first, second, decimal=4)
 
+    def assertCalibratedVariablesAlmostEqual(self, first, second):
+        """Overriding of the assertArrayAlmostEqual method to check whether
+        array are matching to 4 decimal places. This is specifically
+        for use in assertions following applying the EMOS coefficients,
+        in order to calibrate the chosen variables. This is justified
+        based on the default tolerance of the minimisation using the
+        Nelder-Mead algorithm of 0.0001, so that minimisations on different
+        machines would only be aiming to match to 4 decimal places.
+        Args:
+            first (numpy.ndarray):
+                First array to compare.
+            second (numpy.ndarray):
+                Second array to compare.
+         """
+        self.assertArrayAlmostEqual(first, second, decimal=4)
```

Comment:
I get the motivation for recording why we are only testing to 4 decimal places, but this function is identical to the one above, and both could as simply be achieved by using the "decimal=4" argument in the tests themselves, slimming down this helper_functions file. This would also make Tom's life easier as he attempts to wean us off of using the ArrayAlmostEqual functionality provided by iris.

@fionaRust Any thoughts? Could we achieve this better by adding pages to the sphinx documentation that summarise any peculiarities of a set of unit tests (i.e. why we only care about precision of 4 decimal places)?